checkQC/run_type_recognizer.py

```diff
@@ -29,6 +29,28 @@ def __init__(self, config, runfolder):
         with open(os.path.join(self._runfolder, "RunInfo.xml")) as f:
             self._run_info = xmltodict.parse(f.read())
 
+        with open(self._find_run_parameters_xml()) as f:
+            self._run_parameters = xmltodict.parse(f.read())
+
+    def _find_run_parameters_xml(self):
+        first_option = os.path.join(self._runfolder, "RunParameters.xml")
```

Comment:
Looks good, but just a suggestion for a more generic solution (here I assume that any case combination is fine):

```
files = [os.path.join(self._runfolder, f) for f in os.listdir(self._runfolder)
         if os.path.isfile(f) and f.lower() == 'runparameters.xml']
if len(files) == 1:
  return files[0]
else:
  raise ...
```

Side note: In `clarity-ext` we actually have a helper for this, modelled on a very popular built-in in .NET. Equipped with that helper, this style makes much more sense (since the code becomes very clear, methinks):
```
files = [os.path.join(self._runfolder, f) for f in os.listdir(self._runfolder)
         if os.path.isfile(f) and f.lower() == 'runparameters.xml']
return utils.single(files)  # Raises if it's not exactly one.
```